### PR TITLE
Issues 2243

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR for issues #2243 

## How Has This Been Tested?
Tested By chaining the status of the donation from the bulk edit

## Screenshots (jpeg or gifs if applicable):
Singular Message Image:
image

Plural Message Image:
image

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/admin-actions.php [hunk 1]
```diff
@@ -175,6 +175,28 @@ function _give_register_admin_notices() {
...
+				case 'set-status-revoked' :
+				case 'set-status-failed' :
+				case 'set-status-cancelled' :
+				case 'set-status-abandoned' :
+				case 'set-status-preapproval' :
+					Give()->notices->register_notice( array(
+						'id'          => 'bulk_action_status_change',
+						'type'        => 'updated',
+						'description' => _n(
+							'Donation status updated successfully',
```
Comment (ravinderk) [inline includes/admin/admin-actions.php]: @raftaar1191 Where is the colon (.)

Comment (raftaar1191) [reply includes/admin/admin-actions.php]: Thanks @ravinderk For pointing this out

File: includes/admin/tools/data/class-give-tools-delete-test-transactions.php [hunk 2]
```diff
@@ -65,61 +63,10 @@ public function get_data() {
...
-
-						$sql[] = "DELETE FROM $wpdb->posts WHERE id IN ($ids)";
-						$sql[] = "DELETE FROM $wpdb->postmeta WHERE post_id IN ($ids)";
-						$sql[] = "DELETE FROM $wpdb->comments WHERE comment_post_ID IN ($ids)";
-						$sql[] = "DELETE FROM $wpdb->commentmeta WHERE comment_id NOT IN (SELECT comment_ID FROM $wpdb->comments)";
-						break;
-				}
-
+				// Delete the main payment.
+				give_delete_donation( intval( $item['id'] ) );
```
Comment (ravinderk) [inline includes/admin/tools/data/class-give-tools-delete-test-transactions.php]: @raftaar1191 use `absint`

File: includes/admin/admin-actions.php [hunk 3]
```diff
@@ -189,8 +189,8 @@ function _give_register_admin_notices() {
 						'id'          => 'bulk_action_status_change',
 						'type'        => 'updated',
 						'description' => _n(
-							'Donation status updated successfully',
-							'Donations status updated successfully',
+							'Donation status updated successfully.',
+							'Donations status updated successfully.',
```
Comment (DevinWalker) [inline includes/admin/admin-actions.php]: @raftaar1191 this should be:

> Donation statuses updated successfully.

File: includes/admin/admin-actions.php (hunk 1 above)
Comment (DevinWalker) [reply includes/admin/admin-actions.php]: Period, not colon.

File: includes/admin/admin-actions.php (hunk 3 above)
Comment (raftaar1191) [reply includes/admin/admin-actions.php]: Done @DevinWalker

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk /

This PR is ready for review and Merge

**Pull request merged**